common/buildings/wc_special_buildings.txt

```diff
@@ -315,6 +315,30 @@ ruined_palace_01 = {
 	flag = travel_point_of_interest_martial
 }
 
+ruined_palace_02 = { 
+	type_icon = "icon_structure_lordaeron_palace.dds"
+
+	levy = 150
+	province_modifier = {
+		development_growth = 0.1
+		travel_danger = 10
+		fort_level = 2
+	}
+	character_modifier = {
+		intrigue_per_prestige_level = 1
+		dread_gain_mult = 0.1
+		knight_limit = 1
```

Comment:
```suggestion
		dread_gain_mult = 0.2
		knight_limit = 2
```
make it a little stronger. Other way not worth it to build

Reply:
i already buffed it a lot and it's in the balance sheet. also you get good modifiers and religious conversion effects down the road so i think it is more than worth it. go ahead and commit your suggestion if you feel it's not enough : ]

Reply:
Balance sheet consider probably the first level. This is the second, you spend prestige, gold, time and penalty modifiers besides that. And all you get is 2 forts, and 5 scheme resistance. Idk. Well, if you think its ok then ok.